wmfdata/mariadb.py

```diff
@@ -139,19 +139,12 @@ def run(
         # then turn them into a data frame; this would let us avoid using
         # Pandas's complex SQL machinery.
         for command in commands:
-            try:
```

Comment:
Looks like you accidentally dropped this `try`.

Reply:
oopsydoodles; that's what I get for coding when I can't execute. That's addressed now that I have the proxy configured correctly on the stat host. Thanks!